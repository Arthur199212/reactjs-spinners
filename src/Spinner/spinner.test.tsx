import React from 'react'
import renderer from 'react-test-renderer'
import { Spinner } from '.'

test('Component renders correctly', () => {
  const component = renderer.create(<Spinner />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
