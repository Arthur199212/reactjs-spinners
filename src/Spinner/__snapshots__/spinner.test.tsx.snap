// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div
  className="lds-spinner"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
